.changeset/fluffy-eels-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-codegen/typescript-apollo-angular': major
```

Comment:
I think this should be minor but I didn't get option to do it minor

Reply:
Thank you for the PR!
You can manually change it to minor.